pages/common/jdupes.md

```diff
@@ -19,7 +19,7 @@
 
 `jdupes --delete --recurse {{directory}}`
 
-- Search multiple directories, and follow subdirectores under directory2, not directory1:
+- Search multiple directories and follow subdirectores under directory2, not directory1:
```

Comment:
For some reason, directory2 and directory1 appear odd here. Is there a better alternative to be put in their place?

Reply:
Not sure I'm seeing the oddness.